core/src/main/scala/org/apache/spark/jdbc/ConnectionConf.scala

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright (c) 2016 SnappyData, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you
+ * may not use this file except in compliance with the License. You
+ * may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied. See the License for the specific language governing
+ * permissions and limitations under the License. See accompanying
+ * LICENSE file.
+ */
+package org.apache.spark.jdbc
+
+import java.io.{IOException, ObjectOutputStream}
+
+import scala.StringBuilder
+
+import org.apache.spark.sql.SnappyContext
+import org.apache.spark.sql.execution.columnar.ExternalStoreUtils
+import org.apache.spark.sql.execution.columnar.ExternalStoreUtils.CaseInsensitiveMutableHashMap
+import org.apache.spark.sql.sources.ConnectionProperties
+import org.apache.spark.util.Utils
+
+import scala.collection.mutable
+
+class ConnectionConf(val connProps: ConnectionProperties) extends Serializable {
```

Comment:
This can be taken in a separate PR. Those changes will unnecessary crowd this PR , which is for a specific purpose. Will file a JIRA ticket to track .